### PR TITLE
CI, conventions and naming

### PR DESCRIPTION
**Pull request by vespos**
Goal: clean up repo, better CI, better file names (more in line with Typhos and other existing tools), follow some UI standards

To do:

- [x] flake8
- [x] setup repo according to PCDS standards
- [x] rename repo and widget names for a general LCLS use

### CODE REVIEW

**Comment by vespos** [comment]
@ZLLentz I think I would need a little bit of help with the conda errors. Do you think we'd want that package as part of our conda env?

**Comment by ZLLentz** [comment]
It might make sense to edit this file which points to the "do everything" workflow:
https://github.com/pcdshub/ecs-pydm-ui/blob/2c2fdac46c729e475e63f7117d3e6384781ea6d4/.github/workflows/standard.yml#L12
And switch to e.g. one of the sub-workflows that this calls, for example just the pre-commit checks:
https://github.com/pcdshub/pcds-ci-helpers/blob/master/.github/workflows/pre-commit.yml

If we add conda builds later for whatever reason we can restore the more full CI

**Comment by vespos** [comment]
It's working fine now! I think it is ready for review (not an emergency though)

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+[options]
+zip_safe = false
+include_package_data = true
+
+[project.license]
+file = "LICENSE.md"
+
+[tool.setuptools_scm]
+write_to = "lcls_screens/_version.py"
```
Comment (ZLLentz) [inline pyproject.toml:23]: We may need to tweak this slightly if/when we start wanting to distribute this as a package (`lcls_screens` doesn't exist right now). This is something to figure out later though.

**Review by ZLLentz — APPROVED** [review]
This looks solid to me, should be good to merge :+1:

**Pull request merged**